user/templates/templates.md

```diff
@@ -155,22 +155,24 @@ and also that it is not set as the default template.
 
 The procedure for uninstalling a template depends on how it was created.
 
-If the template was originaly created by cloning another template, then you can
+If the template was originally created by cloning another template, then you can
 delete it the same way as you would any other qube. In the Qube Manager,
 right-click on the template and select **Delete qube**. (If you're not sure,
 you can safely try this method first to see if it works.)
```

Comment:
Do you want to mention `qvm-remove`?

Reply:
I'm not sure it would be worth mentioning `qvm-remote`. The documention gives one path that suits every user: using the GUI.

For advanced users, I wonder if a list of Qubes OS specific commands and tooling exist somewhere? This class of user may want to seek that kind of information when dealing with a problem in order to automate it or run in the console. That's why I don't think it's worth a mention here.